src/properties/Messages.ts

```diff
@@ -1,10 +1,22 @@
+import { createIntl, createIntlCache } from 'react-intl';
+
 import { DeepReadonly } from 'ts-essentials';
 import { IntegrationType } from '../types/Integration';
+import { intlHelper } from '@redhat-cloud-services/frontend-components-translations';
+import messages from './DefinedMessages';
+
+const cache = createIntlCache();
+const locale = navigator.language.slice(0, 2);
+const intl = createIntl({
+    // eslint-disable-next-line no-console
+    onError: console.log,
+    locale
+}, cache);
+const intlSettings = { locale };
 
-//Capture some strings we reuse. Possibly use in i18n later?
 const MutableMessages = {
-    appName: 'Notifications',
-    appNameIntegrations: 'Integrations',
+    appName: intlHelper(intl.formatMessage(messages.notifications), intlSettings),
```

Comment:
Maybe we should remove these `appNames` / `appNameIntegrations` and only use the ones in DefinedMessages. Eventually this file will be gone in that case.

Reply:
yeah! absolutely

Reply:
we can duplicate the page location through the id or description fields if desired 🤔

Reply:
You mean the URL? It shouldn't be internationalized anyway, so we might put that on the `Config`. Or in what context/file are you referring ?

Reply:
oooo was referring to the helpfulness of the `MutableMessages` json in Messages.ts, an example of an id could be... integrations-list-title